pages/api/products.js

```diff
@@ -2,15 +2,31 @@ import Shopify, {context} from '@lib/shopify'
 
 export default async (req, res) => {
   // Provide HOST_NAME here just in case it was not provided by env variable
-  // This might occur during the first deploy to Vercel when you don't yet know 
+  // This might occur during the first deploy to Vercel when you don't yet know
   // what domain your app is being hosted on
   Shopify.Context.update({ HOST_NAME: req.headers.host});
 
   const session = await Shopify.Utils.loadCurrentSession(req, res);
-  const client = new Shopify.Clients.Rest(session.shop, session.accessToken);
+  const client = new Shopify.Clients.Graphql(session.shop, session.accessToken);
   // Use `client.get` to request the specified Shopify REST API endpoint, in this case `products`.
```

Comment:
Just spotted this in passing @dan-gamble 

```suggestion
  // Use `client.get` to request the specified Shopify GraphQL API endpoint, in this case `products`.
```